qhexview.cpp

```diff
@@ -626,6 +683,33 @@ int64_t QHexView::pixelToWord(int x, int y) const {
 	return word;
 }
 
+//------------------------------------------------------------------------------
+// Name: updateToolTip
+//------------------------------------------------------------------------------
+void QHexView::updateToolTip() {
+	if(selectedBytesSize() <= 0) {
+		return;
+	}
+
+	auto sb = selectedBytes();
+	auto tooltip = QString("<p style='white-space:pre'>"); //prevent word wrap
+#if 0
+	// Plain formatting with 0x prefix
+	tooltip += QString("<b>Addr:</b> 0x%1 - 0x%2").arg(selectedBytesAddress(), 0, 16)
+												  .arg(selectedBytesAddress() + sb.size(), 0, 16);
```

Comment:
Do you want to include this commented out code? I'm guessing no. If you do, it's got some weird spacing issue on this line.